README.md

```
@@ -1 +1,161 @@
+
+Compare published view files in `resources/views/vendor/`:
+
+```bash
+php artisan vendor-cleanup:view
+```
+
+## Output Categories
+
+Each command categorizes files into four groups:
+
+### MODIFIED (color-coded by % different)
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)